pkg/catalog/diagram.go

```diff
@@ -471,13 +471,14 @@ func (p *Generator) ModuleAsPackages(m *sysl.Module) map[string]*sysl.Module {
 	_, includedProjects := p.getProjectApp(m)
 	for _, key := range SortedKeys(m.GetApps()) {
 		app := m.GetApps()[key]
-		packageName := Attribute(app, "package")
+		packageName, _ := GetAppPackageName(app)
```

Comment:
A little bit confused here, `GetPackageName` already calls GetAppPackageName internally, so do we need to call it here as well? Otherwise, we can also handle the empty case inside `GetPackageName` to simplify the call here?

Reply:
in the next lines there’s a map read with the package name to see if the package has any special attributes.
After this check this variable gets changed to the pretty name